### DISCUSSION TITLE
Latest version missing background/font color change section of settings

### DISCUSSION THREAD

**Discussion by jamesisin** [strawberrymusicplayer/strawberry#1110] [Q&A]
I just updated Strawberry (on Ubuntu Studio (18.04.6) to version 1.0.14) and the section in Settings where one would alter the background and font colors is missing.  

Now I have a stark white application.  I don't need a sun tan.  

Is this happening to others?  

Also, on my Ubuntu laptop (22.04.1) launching Strawberry no longer opens the application window (so I wasn't able to confirm the same behavior) but merely opens the bar applet (which cannot actually play any music).  

Ok, these look like two separate issues.

I found another lappy here running Ubuntu 22.4.1 and Strawberry 1.0.10. Running Ubuntu updates I don't see a new version pending. It is behaving as expected (the main application window launches and my color scheme is intact). Both laptops are running the same version of Ubu and Straw and have different behaviors. Probably something happening on the bad lappy.

The desktop running Ubu Studio has the latest version (1.0.14) and has no color scheme section in Settings --> Appearance (and thus has reverted to meat-space legacy white paper background and black text).

**Comment by jonaski** [comment]
Currently not available, but I plan to add the color options back at some point.
